### DISCUSSION TITLE
Using Deserialize instaed of JsonObject

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/WinUI-Gallery#1270] [General]
I can see in `controlinfodatasource` you are using `jsonobject` for deserializing json file, is there any specific reason for that? Why you are not using `System.Text.JsonSerializer.Deserialize` method?
Using Deserialize method is clean and we can use Async method and mybe faster than `jsonobject`!
I can make a pr for this.
@chingucoding can you answer this question?

**Comment by marcelwgn** [comment]
The reason why it wasn't used initially has to do with the history of the WinUI Gallery, formerly XAML Controls Gallery. This project started out as UWP app, well before the System.Text.JSON namespace was even a thing. Throughout the years the area did not really needed modification and only with the move to WinUI 3 and dropping UWP support, this became a valid option. 

I don't think there is an argument against switching to this API now though, @karkarl what do you think?

**Reply by karkarl** [reply]
@chigucoding nailed the explanation. @ghost1372 if you can make a pr to optimize this, that'll be awesome!

**Reply by ghost1372** [reply]
Tnx, yes i can make a pr

**Comment by ghost1372** [comment]
Hi @chingucoding 
I added a pr 
https://github.com/microsoft/WinUI-Gallery/pull/1276

**Reply by marcelwgn** [reply]
Awesome, I'll take a look later!

**Comment by karkarl** [comment]
Wooo! Fixed with #1276, thanks @ghost1372!
